tests/foreman/maintain/test_backup_restore.py

```diff
@@ -47,22 +46,10 @@
 assert_msg = "Some required backup files are missing"
 
 
-def get_exp_files(sat_maintain, backup_type):
-    if type(sat_maintain) is Satellite:
-        if sat_maintain.is_remote_db():
-            expected_files = BASIC_FILES | REMOTE_SAT_FILES
-        else:
-            expected_files = (
-                BASIC_FILES | OFFLINE_FILES
-                if backup_type == 'offline'
-                else BASIC_FILES | ONLINE_SAT_FILES
-            )
-    else:
```

Comment:
Have you run the other tests in this module to ensure that your changes don't break them? My hunch is that removing this part of the conditional could affect tests in this module that run against Capsules.

Reply:
We can run the backup and restore tests on PRT here and see.

Reply:
It looks like the tests with Capsules did indeed fail, but the errors were all with attaching subscriptions, which at first glance does not appear to be connected to this specific change. What do you think, @Griffin-Sullivan?

Reply:
Yeah I was looking through them. I'm gonna do some local runs tomorrow just to double check. I'm not super worried about the change though. It's just removing a constant that is unnecessary.

Reply:
I just realized that I misread `ONLINE_CAPS_FILES` as `ONLINE_SAT_FILES` in this method and that's why I gutted it. I'm now testing it locally with a fix to see if it will work.